crud.go

```diff
@@ -28,29 +28,29 @@ const (
 	SimulatedVBucketCount = 1024 // Used when hashing doc id -> vbno
 )
 
-// The persistent portion of a lolrus object (the stuff that gets archived to disk.)
-type lolrusData struct {
+// The persistent portion of a Bucket object (the stuff that gets archived to disk.)
+type walrusData struct {
 	LastSeq    uint64                         // Last sequence number assigned
-	Docs       map[string]*lolrusDoc          // Maps doc ID -> lolrusDoc
+	Docs       map[string]*walrusDoc          // Maps doc ID -> walrusDoc
 	DesignDocs map[string]*sgbucket.DesignDoc // Stores source form of design docs
 }
 
 // Simple, inefficient in-memory implementation of Bucket interface.
 // http://ihasabucket.com
-type lolrus struct {
+type Bucket struct {
```

Comment:
I'm also a bit sad to see lolrus get refactored to Bucket - it takes away some of walrus's personality, I think, which always seemed reasonable for an internal library like this one.  You might be able to talk me into the change, but I'm curious to hear the justification.